_health-care/_js/_form.js

```diff
@@ -18,17 +18,60 @@ var HealthApp = (function() {
     var dateString = "";
 
     if (date.getMonth() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += (date.getMonth() + 1) + "/";
 
     if (date.getDate() < 10) {
-      dateString += "0"
+      dateString += "0";
     }
     dateString += date.getDate() + "/";
 
     return dateString + date.getFullYear();
   }
+  function extractMonetaryValueTransform(formData, arg) {
+    return Number(extractTransform(formData, arg)).toFixed(2).toString();
+  }
+  function validatePresence(input) {
+    var valid = true;
+
+    if (input.value === '') {
+      valid = false;
+    }
+
+    return valid;
+  }
+  function validateMonetaryValue(input) {
+    var valid = true;
+    var amount = input.value;
+
+    if (/[$a-zA-Z,]+/.test(amount)) {
+      valid = false;
+    }
+
+    return valid;
+  }
+  function validate(input) {
+    var valid = true;
+
+    $(input).removeClass('error');
```

Comment:
Since we have modernizr loaded, I think that means we can use the html5 variants for these.

```
input.classList.remove('error');
input.classList.add('error');
```

https://davidwalsh.name/classlist for reference.

Samething with input.dataSet.  I don't think it's worth your time to muck with it in this PR, but maybe create a bug and assign to me?